### PR TITLE
CONTROLLERS: Delete `Time`

### LINKED ISSUE #229: CONTROLLERS: Delete `Time`

### PR DESCRIPTION
**Pull request by MirzayevDeveloper**
closes #229

### CODE REVIEW

File: Tarteeb.Api/Controllers/TimesController.cs [hunk 1]
```diff
@@ -73,5 +74,43 @@ public ActionResult<IQueryable<Time>> GetAllTimes()
                 return InternalServerError(timeServiceException.InnerException);
             }
         }
+
+        [HttpDelete("{timeId}")]
+        public async ValueTask<ActionResult<Time>> DeleteLanguageByIdAsync(Guid timeId)
```
Comment (ShohruhUzDev) [inline Tarteeb.Api/Controllers/TimesController.cs:79]: Language emas Time buladi

Comment (MirzayevDeveloper) [reply Tarteeb.Api/Controllers/TimesController.cs]: resolved.

**Review by ShohruhUzDev — APPROVED** [review]

**Comment by MirzayevDeveloper** [comment]
image

**Comment by Javlonbek-dev** [comment]
@MirzayevDeveloper  404 Xatolikni ham Screen qilib tashlang

**Review by ElbekDeveloper — APPROVED** [review]

**Pull request merged**